bin/reth/src/commands/import.rs

```diff
@@ -215,7 +215,7 @@ mod tests {
     fn parse_common_import_command_chain_args() {
         for chain in SUPPORTED_CHAINS {
             let args: ImportCommand = ImportCommand::parse_from(["reth", "--chain", chain, "."]);
-            assert_eq!(args.chain.chain, chain.parse().unwrap());
+            assert_eq!(&args.chain.chain, &chain.parse::<alloy_chains::Chain>().unwrap());
```

Comment:
this shouldn't be required

Reply:
.parse() is unable to infer type here, due which code is not compiling, @Rjected mentioned above to use this solution, if there is any other way you can surely tell me